src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,45 @@
 import React from 'react';
+import './PostInfo.scss';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+import { Post } from '../../types/Post';
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+
+type Props = {
+  post: Post
+};
+
+export const PostInfo: React.FC<Props> = ({ post }) => {
+  const {
+    title,
+    body,
+    user,
+    comments,
+  } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{title}</h3>
+
+        <p>
+          {' Posted by  '}
```

Comment:
You don't need these parentheses here
```suggestion
          Posted by
```